Edit Functionality

snehabanda30 opened the pull request:
Added in the edit functionality. After the user signs up, it brings to the site where the user is logged in with current username. However, I am still unable to get the patch request to work, which I will continue to try working on.

snehabanda30 committed:
Edit Functionality

Added in the edit functionality. After the user signs up, it brings to the site where the user is logged in with current username. However, I am still unable to get the patch request to work, which I will continue to try working on.

Victor-M-Giraldo commented:
Think I found a fix for the issue that required you to use the insecure prototype access package. This way we can also just get rid of it.

https://stackoverflow.com/questions/59690923/handlebars-access-has-been-denied-to-resolve-the-property-from-because-it-is

snehabanda30 committed:
Made changes to the edit

It is fully functional now! Please let me know if there is any issues with my code.

snehabanda30 commented:
Not sure if there is an hbs file at the moment.
